src/app/pages/send/send-crypto-asset-preview/components/send-crypto-asset-preview.layout.tsx

```diff
@@ -0,0 +1,21 @@
+import { Flex, StackProps } from '@stacks/ui';
+
+import { CENTERED_FULL_PAGE_MAX_WIDTH } from '@app/components/global-styles/full-page-styles';
+
+export function SendCryptoAssetPreviewLayout({ children }: StackProps) {
+  return (
+    <Flex
+      alignItems="center"
+      flexGrow={1}
+      flexDirection="column"
+      justifyContent="start"
+      mb="loose"
+      minHeight={['70vh', '90vh']}
+      maxWidth={['unset', 'unset', CENTERED_FULL_PAGE_MAX_WIDTH]}
```

Comment:
isn't this the same as `width=100%` if it's within that container anyway?

Reply:
Let me check on these, might be true but I'm not sure the container sets this?

Reply:
This isn't defined in the route container bc some pages don't use this (ex. onboarding).